src/styles/content.scss

```diff
@@ -254,6 +254,18 @@ table.os-raise-horizontaltable {
   row-gap: 5rem;
 }
 
+.os-raise-d-flex-nowrap {
+  display: flex;
```

Comment:
Minor: I know it's the default, but maybe it would be nice to explicitly say `flex-wrap: nowrap`?

Reply:
Is the suggestion to explicitly state `flex-wrap: nowrap` because we have a flex class that does have wrap? The reason I ask is because I think there may be additional defaults that are a part of `display: flex` (like flex-direction) and want to make sure the intent isn't to identify all defaults.

Reply:
Yeah, it was mostly just to aid a bit of readability. You make a good point, though, that maybe the specificity of just the one may make things muddier. I rescind this comment :joy: .